services/users.js

```diff
@@ -447,6 +451,48 @@ module.exports = class UsersService {
   }
 
   /**
+   * Suspend a user. It changes the status to BANNED and canEditName to True.
+   * @param  {String}   id   id of a user
+   * @param  {Function} done callback after the operation is complete
+   */
+  static suspendUser(id, message) {
+    return UserModel.update({
```

Comment:
Can use a `findAndModify` instead of `update` to perform both the `update/find` at the same time.